crates/boundless-market/src/contracts/mod.rs

```diff
@@ -523,6 +523,18 @@ impl Requirements {
     pub fn with_selector(self, selector: FixedBytes<4>) -> Self {
         Self { selector, ..self }
     }
+
+    /// Set the selector for an unaggregated proof.
+    ///
+    /// This will set the selector to the appropriate value based on the current environment.
+    /// In dev mode, the selector will be set to `FakeReceipt`, otherwise it will be set
+    /// to `Groth16V1_2`.
+    pub fn with_unaggregated_proof(self) -> Self {
+        match is_dev_mode() {
+            true => Self { selector: FixedBytes::from(Selector::FakeReceipt as u32), ..self },
+            false => Self { selector: FixedBytes::from(Selector::Groth16V1_2 as u32), ..self },
+        }
+    }
```

Comment:
I've added this utility so that, since we can only support one zkvm version at the moment, it's easier for the requestor to use this rather than looking for the specific selector

Reply:
It's also easier in tests as it automatically select the Fake receipt if in dev mode.